### PR TITLE
Release notes for week of Jan 13

### PR DESCRIPTION
**Pull request by srpearce**
## [DOC-3547](https://openedx.atlassian.net/browse/DOC-3547)

Add a description of your changes with links to any relevant material.

### Date Needed (optional)

18 January

### Reviewers

Possible roles follow. The PR submitter checks the boxes after each reviewer finishes and gives :+1:. 

- [ ] Subject matter expert: @amangano-edx 
- [ ] Subject matter expert: @cahrens 
- [ ] Doc team review (sanity check, copy edit, or dev edit?): @catong 

### Testing

- [ ] Ran ./run_tests.sh without warnings or errors

### Post-review

- [ ] Add a comment with the description of this change or link this PR to the next release notes task.
- [ ] Squash commits

### CODE REVIEW

File: en_us/release_notes/source/2017/studio/studio_2017-01-13.rst [hunk 1]
```diff
@@ -0,0 +1,7 @@
+* The problem template for Custom JavaScript display and grading problems (also
+  called custom JavaScript problems or JS input problems) has been updated.
+  :jira:`TNL-5893`
+
+* On unit pages in Studio, the guidance in the **Unit Location** panel now
+  instructs course teams to use ``/jump_to_id/<location ID>`` formatting to
+  create links to units.
```
Comment (catong) [inline en_us/release_notes/source/2017/studio/studio_2017-01-13.rst]: An example of using the link formatting would be helpful. Maybe at least in the portal post?

File: en_us/release_notes/source/2017/2017-01-13.rst [hunk 2]
```diff
@@ -0,0 +1,36 @@
+#################################
+7 January through 13 January 2017
```
Comment (catong) [inline en_us/release_notes/source/2017/2017-01-13.rst]: We could go back to the "Week ending 13 January" wording? The previous release was an anomaly spanning more than a week, which was the only reason I called it something different. Going back to "Week ending" wording might also reinforce that release notes will be released more regularly going forward.

File: en_us/release_notes/source/2017/documentation/docs_2017-01-13.rst [hunk 3]
```diff
@@ -0,0 +1,5 @@
+The documentation for JS input problems has been updated to reflect the new
+``title`` attribute inside the ``jsinput`` tag. For more information, see
```
Comment (catong) [inline en_us/release_notes/source/2017/documentation/docs_2017-01-13.rst]: The doc update reflects the title attribute as well as the new template. 
I would suggest using the same problem type name "Custom JavaScript Display and Grading Problem (also known as...)" in both the doc item and the Studio item.

File: en_us/release_notes/source/2017/studio/studio_2017-01-13.rst [hunk 4]
```diff
@@ -0,0 +1,7 @@
+* The problem template for Custom JavaScript display and grading problems (also
```
Comment (catong) [inline en_us/release_notes/source/2017/studio/studio_2017-01-13.rst]: Initial caps also on "Display" and "Grading": "Custom JavaScript Display and Grading Problem" is the problem type's name.

File: en_us/release_notes/source/2017/documentation/docs_2017-01-13.rst [hunk 5]
```diff
@@ -0,0 +1,5 @@
+The documentation for JS input problems has been updated to reflect the new
```
Comment (cahrens) [inline en_us/release_notes/source/2017/documentation/docs_2017-01-13.rst]: Note that the documentation has also been largely rewritten due to the new example problem.

**Comment by cahrens** [comment]
@srpearce LGTM, though you may want to call out the updated JSInput doc (beyond just the title attribute).

**Review by catong — COMMENTED** [review]
A few comments including the release title ("Week ending...")

File: en_us/release_notes/source/2017/2017-01-13.rst (hunk 2 above)
Comment (srpearce) [reply en_us/release_notes/source/2017/2017-01-13.rst]: Oops - I changed this in the other files. Will fix...

File: en_us/release_notes/source/2017/lms/lms_2017-01-13.rst [hunk 6]
```diff
@@ -0,0 +1,7 @@
+* When learners select the **Support** option in the LMS to contact learner
+  support, learners can specify the course that is the subject of their
+  question or comment. :jira:`ECOM-5936`
```
Comment (srpearce) [inline en_us/release_notes/source/2017/lms/lms_2017-01-13.rst:3]: @amangano-edx Can you let me know if the above is accurate?

Comment (amangano-edx) [reply en_us/release_notes/source/2017/lms/lms_2017-01-13.rst:3]: Looks good to me.

**Comment by catong** [comment]
👍

**Pull request merged**